### PR TITLE
Add help and fix bugs

### PR DESCRIPTION
**Pull request by duartecoelhomovai**

### CODE REVIEW

File: mobros/commands/ros_install_runtime_deps/install_deps_executer.py [hunk 1]
```diff
@@ -382,15 +384,28 @@ def execute(self, args):
...
             nargs="+",
             default=[],
+            help="List of packages to be installed, just like apt. It can contain the specific version of it <name>=<version>"
         )
         parser.add_argument(
             "--upgrade-installed",
             required=False,
             action="store_true",
             dest="upgrade_installed",
+            help="Don't mind the versions that are installed, use the latest avaiable in apt cache."
```
Comment (AlexFernandes-MOVAI) [inline mobros/commands/ros_install_runtime_deps/install_deps_executer.py:398]: latest avai-l-able

File: mobros/handler.py [hunk 2]
```diff
@@ -27,63 +26,54 @@
...
-        required=False,
-        action="store_true",
-    )
     # executor arguments
-    for executer in executors.values():
-        executer.add_expected_arguments(parser)
+    # pylint: disable=W0702
+    try:
+        ns, _ = pre_parser.parse_known_args()
+    except:
```
Comment (AlexFernandes-MOVAI) [inline mobros/handler.py:46]: use ``except Exception:``

File: mobros/utils/version_utils.py [hunk 3]
```diff
@@ -327,3 +351,48 @@ def create_version_rule(operation, version, from_str):
         "version": version,
         "from": from_str
     }
+
+# pylint: disable=R0911
+def version_impacts_version_rules(compraring_version, rules):
+    """ Checks if a giver version is compatible with a list of version rules
+
+    Args:
+        compraring_version (str): package version
```
Comment (AlexFernandes-MOVAI) [inline mobros/utils/version_utils.py:360]: compraring_version should be comparing_version

File: tests/test_handler.py [hunk 4]
```diff
@@ -7,22 +7,21 @@
...
 def mock_add_expected_arguments(parser):
-    parser.add_argument("--dummy_arg")
-
+    return [None, None]
 
 argeparse_executor_build = argparse.Namespace(
-    command="build", workspace="DUMMY_PATH", mode="debug"
+    command="build", workspace="DUMMY_PATH", mode="debug", h=False
 )
 argeparse_executor_pack = argparse.Namespace(
```
Comment (AlexFernandes-MOVAI) [inline tests/test_handler.py:15]: argeparse_executor_pack should be argparse_executor_pack

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Comment by duartecoelhomovai** [comment]
Added version printed on all commands

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Comment by AlexFernandes-MOVAI** [comment]
> Kudos, SonarCloud Quality Gate passed!    [Quality Gate passed](https://sonarcloud.io/dashboard?id=MOV-AI_mobros-build-system&pullRequest=92)
> 
> [Bug](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=BUG) [C](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=BUG) [1 Bug](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=CODE_SMELL) [C](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=CODE_SMELL) [236 Code Smells](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=CODE_SMELL)
> 
> [](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_coverage&view=list) [81.3% Coverage](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_duplicated_lines_density&view=list)

Please check https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=92&id=MOV-AI_mobros-build-system&open=AYi2Ahky3bwbCtHm1sCD

**Comment by duartecoelhomovai** [comment]
> > Kudos, SonarCloud Quality Gate passed!    [Quality Gate passed](https://sonarcloud.io/dashboard?id=MOV-AI_mobros-build-system&pullRequest=92)
> > [Bug](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=BUG) [C](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=BUG) [1 Bug](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=BUG) [Vulnerability](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=VULNERABILITY) [A](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=VULNERABILITY) [0 Vulnerabilities](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=VULNERABILITY) [Security Hotspot](https://sonarcloud.io/project/security_hotspots?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=SECURITY_HOTSPOT) [A](https://sonarcloud.io/project/security_hotspots?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=SECURITY_HOTSPOT) [0 Security Hotspots](https://sonarcloud.io/project/security_hotspots?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=SECURITY_HOTSPOT) [Code Smell](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=CODE_SMELL) [C](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=CODE_SMELL) [236 Code Smells](https://sonarcloud.io/project/issues?id=MOV-AI_mobros-build-system&pullRequest=92&resolved=false&types=CODE_SMELL)
> > [](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_coverage&view=list) [81.3% Coverage](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_coverage&view=list) [](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_duplicated_lines_density&view=list) [0.0% Duplication](https://sonarcloud.io/component_measures?id=MOV-AI_mobros-build-system&pullRequest=92&metric=new_duplicated_lines_density&view=list)
> 
> Please check https://sonarcloud.io/project/issues?resolved=false&types=BUG&pullRequest=92&id=MOV-AI_mobros-build-system&open=AYi2Ahky3bwbCtHm1sCD

Bug decided as false positive

**Pull request merged**